dev/drivers/scripts/global_ens/plots/jevs_global_ens_wave_grid2obs_plots.sh

```diff
@@ -13,37 +13,32 @@ set -x
 #%include <head.h>
 #%include <envir-p1.h>
 
-export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+#export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/gitworkspace/EVS
 
 ############################################################
 # read version file and set model_ver
 ############################################################
 versionfile=$HOMEevs/versions/run.ver
 . $versionfile
-export evs_ver=$evs_ver
+#export evs_ver=$evs_ver
 export model_ver=$gefs_ver
 export obsproc_ver=$obsproc_ver
```

Comment:
You can eliminate line 24: `#export evs_ver=$evs_ver`
You can also eliminate line 26: `export obsproc_ver=$obsproc_ver`. $obsproc_ver is set in run.ver

Reply:
Done! But do you want this to extend to all instances within all wave drivers?

Reply:
Applied changes to all wave drivers (prep, stats, and plots).